docs/experiments/generic-pattern-matching.md

```diff
@@ -1,38 +1,77 @@
 ---
 append_help_link: true
-description: "Semgrep can match matches generic patterns in languages that it doesn’t yet support. You can use generic pattern matching for languages that don’t yet have a parser, configuration files, and other structured data, such as HTML or XML. Generic pattern matching is experimental."
+description: "Semgrep can match matches generic patterns in languages that it doesn’t yet support. You can use generic pattern matching for languages that don’t yet have a parser, configuration files, and other structured data such as XML. Generic pattern matching is experimental."
 ---
 
 # Generic pattern matching
+
+<!-- If you ever need to replace the examples below, a good way is to look
```

Comment:
Thank you! This will help us maintain this. :)

Reply:
Yes, thank you!